crowbar_framework/app/views/barclamp/_node_selector.html.haml

```diff
@@ -35,7 +35,7 @@
             %ul.clusters
               - @service_object.available_clusters.sort.each do |cluster_name, role|
                 - cluster_barclamp, cluster_proposal = @service_object.cluster_get_barclamp_and_proposal(cluster_name)
-                %li.form-control.input-sm.available{ "data-draggable" => "true", "data-id" => cluster_name, "data-alias" => cluster_proposal, "data-cluster" => "true", "data-admin" => "false", :title => cluster_proposal }
+                %li.form-control.input-sm.available{ "data-draggable" => "true", "data-id" => cluster_name, "data-alias" => cluster_proposal, "data-cluster" => "true", "data-admin" => "false", "data-platform" => "cluster", "data-platform-version" => "cluster", :title => cluster_proposal }
```

Comment:
IDK, is added into the NodeList insertNode method, and I provide and isomorphic data structure for both clustered and single node.  Maybe an empty value, instead of a random one like 'cluster' is more correct here?